src/nvim/help.c

```diff
@@ -653,35 +652,6 @@ void fix_help_buffer(void)
     curbuf->b_ro_locked--;
   }
 
-  if (!syntax_present(curwin)) {
-    bool in_example = false;
-    for (linenr_T lnum = 1; lnum <= curbuf->b_ml.ml_line_count; lnum++) {
-      char *line = ml_get_buf(curbuf, lnum);
-      const size_t len = strlen(line);
-      if (in_example && len > 0 && !ascii_iswhite(line[0])) {
-        // End of example: non-white or '<' in first column.
-        if (line[0] == '<') {
-          // blank-out a '<' in the first column
-          line = ml_get_buf_mut(curbuf, lnum);
-          line[0] = ' ';
-        }
-        in_example = false;
-      }
-      if (!in_example && len > 0) {
-        if (line[len - 1] == '>' && (len == 1 || line[len - 2] == ' ')) {
-          // blank-out a '>' in the last column (start of example)
-          line = ml_get_buf_mut(curbuf, lnum);
-          line[len - 1] = ' ';
-          in_example = true;
-        } else if (line[len - 1] == '~') {
-          // blank-out a '~' at the end of line (header marker)
-          line = ml_get_buf_mut(curbuf, lnum);
-          line[len - 1] = ' ';
-        }
-      }
-    }
-  }
-
   // In the "help.txt" and "help.abx" file, add the locally added help
```

Comment:
Do we even have `help.abx` files? The longer I look at this function, the more I want to burn it down...

Reply:
Plugin translations

Reply:
Oh, it's for translations....